plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterClearButton.php

```diff
@@ -54,15 +54,15 @@ protected function render( $attributes, $content, $block ) {
 
 		$wrapper_attributes = get_block_wrapper_attributes(
 			array(
-				'data-wc-bind--hidden' => '!state.hasSelectedFilters',
-				'data-wc-interactive'  => wp_json_encode( array( 'namespace' => $block->context['filterData']['parent'] ), JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_QUOT | JSON_HEX_AMP ),
+				'data-wp-bind--hidden' => '!state.hasSelectedFilters',
+				'data-wp-interactive'  => wp_json_encode( array( 'namespace' => $block->context['filterData']['parent'] ), JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_QUOT | JSON_HEX_AMP ),
```

Comment:
namespace

Reply:
So the namespace is being passed in and not its own so we can't change this.

Reply:
Just do:

```suggestion
				'data-wp-interactive'  => $block->context['filterData']['parent'],
```

Reply:
@roykho Just this one left ^

Reply:
@samueljseay should have been done. Did you pull latest?

Reply:
Ah sorry I just saw you resolved it before this comment was made :)